_posts/2024-06-11-quarkus-in-a-foundation.adoc

```diff
@@ -0,0 +1,62 @@
+
+== Vision for Quarkus
+As Quarkus adoption increases, we aim to establish Quarkus as the de-facto standard for new Java-based applications. We are excited to welcome more developers and vendors from across the software ecosystem, inviting them to be part of this dynamic journey.
+
+== Enhancing Governance and Execution
+We are committed to improving our governance and execution by enabling more transparency and participation. Over the coming weeks and months, we will roll out initiatives designed to:
+
+Increase Transparency:: We will provide more visibility into our decision-making processes, development roadmaps, and project status updates to the community.
+Enhance Participation:: We will create more opportunities for community members to get involved in project governance, such as contributing to working efforts, joining meetings/chats, and participating in design discussions.
+Improve Communication:: We will establish better channels for feedback and dialogue, ensuring that community voices are heard and considered in the project’s evolution.
+
+== The Next Step: Moving to a Foundation
+To further this goal, we are proposing an exciting new chapter: moving the Quarkus project to a foundation. This move will bolster our commitment to open-source development and inspire greater community participation.
+
+We believe transitioning Quarkus to a foundation will help:
+
+* Accelerate adoption rate.
+* Enhance transparency and foster collaboration.
+* Encourage multi-vendor participation and execution.
+
+== Support and Alignment with Red Hat Values
+
+Red Hat business leaders are fully behind this move. Red Hat is dedicated to participating in and supporting vendor-neutral collaboration projects, such as the Linux kernel, Kubernetes, and OpenJDK. We seek the same for Quarkus.
+
+== Community Feedback
+
+Discussions with Red Hat and non-Red Hat community members indicate strong support for this move. The feedback received so far highlights Quarkus' maturity and the need to improve its development openness.
+
+== Criteria for Choosing the Right Foundation
+
+We have identified several key criteria for selecting a foundation:
+
+Maintain Delivery Pace:: Preserve Quarkus fast pace of delivery, innovation, and experimentation.
+Visibility and Recognition:: Ensure Quarkus remains visible and recognizable within a foundation's potentially larger portfolio of projects.
+Flexibility Beyond Cloud Native:: Recognize that Quarkus supports a wide variety of use cases, from monoliths to edge deployments.
+Independent Technology Choices:: Ensure Quarkus can make decisions based on technical merits, not foundation-imposed options.
+Licence Flexibility:: Provide flexibility in using OSI-approved licences for the Quarkus platform, its extensions and its dependencies.
```

Comment:
I know it's a bit redundant but I'm not sure everyone knows what `OSI` is and I think given the context, it's important we mention we are committed to Open Source.
```suggestion
Licence Flexibility:: Provide flexibility in using OSI-approved Open Source licences for the Quarkus platform, its extensions and its dependencies.
```